### PR TITLE
Recognize .webp as embeddable image

### LINKED ISSUE #170: Emanote doesn't render WebP
I link a [webp](https://en.wikipedia.org/wiki/WebP) image like [this](https://note.ema.srid.ca/demo/embed#images) and expect it to render:

```
![[series.squid-game.webp]]
![Img](../assets/series.squid-game.webp)
```
This is the result:

2021-10-08_13-51-emanote-webp-render

It appears to me as if Emanote doesn't treat `.webp` as images but as files instead.

Hovering over the first link (with [Imagus](https://addons.mozilla.org/en-US/firefox/addon/imagus/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search)) correctly displays the image. Same with clicking on it. I therefore believe that I didn't mistype the path or something.

To test: [This](https://upload.wikimedia.org/wikipedia/commons/a/a1/Johnrogershousemay2020.webp) is a webp image.

### PR DESCRIPTION
**Pull request by j-steinbach**
Resolves https://github.com/srid/emanote/issues/170.

----

Note: I don't know Haskell so this isn't tested. I also don't really know how to work with git/github.

### CODE REVIEW

**Comment by j-steinbach** [comment]
Tested it, this doesn't run

```
src/Emanote/Pandoc/Renderer/Embed.hs:93:5-10: error:
    • No instance for (IsString (t0 -> Text))
        arising from the literal ‘".bmp"’
        (maybe you haven't applied a function to enough arguments?)
    • In the expression: ".bmp"
      In the expression: ".bmp" ".webp"
      In the expression: [".jpg", ".jpeg", ".png", ".svg", ....]
   |
93 |     ".bmp"
   |     ^^^^^^
src/Emanote/Pandoc/Renderer/Embed.hs:94:5-11: error:
    • Ambiguous type variable ‘t0’ arising from the literal
‘".webp"’
      prevents the constraint ‘(IsString t0)’ from being solved.
      Probable fix: use a type annotation to specify what ‘t0’
should be.
      These potential instances exist:
        instance IsString a => IsString (Identity a)
          -- Defined in ‘Data.String’
        instance IsString ByteString
          -- Defined in ‘Data.ByteString.Internal’
        instance IsString ShortByteString
          -- Defined in ‘Data.ByteString.Short.Internal’
        ...plus five others
        ...plus 23 instances involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In the first argument of ‘".bmp"’, namely ‘".webp"’
      In the expression: ".bmp" ".webp"
      In the expression: [".jpg", ".jpeg", ".png", ".svg", ....]
   |
94 |     ".webp"
   |     ^^^^^^^
   ```

File: src/Emanote/Pandoc/Renderer/Embed.hs [hunk 1]
```diff
@@ -91,6 +91,7 @@ imageExts =
     ".svg",
     ".gif",
     ".bmp"
```
Comment (srid) [inline src/Emanote/Pandoc/Renderer/Embed.hs]: You are missing a `,` after the bmp line.

In Haskell, a "list" is a comma separated list of values (the value here being a quoted string). When adding an item to the list you just naturally add an extra comma.

http://learnyouahaskell.com/starting-out#an-intro-to-lists

**Review by srid — CHANGES_REQUESTED** [review]
@j-steinbach Hey, congrats on your first Haskell contribution!

Could you also change the version number? 

1. Go to https://github.com/j-steinbach/emanote/blob/webp/emanote.cabal
2. Click "Edit"
3. Change `0.3.5.2` in line 3 to `0.3.6.0`.

**Review by srid — APPROVED** [review]

**Comment by j-steinbach** [comment]
Thanks for the guidance. I guess this (and the issue) can be closed now?

**Comment by srid** [comment]
PRs should be closed only after merging.

**Comment by j-steinbach** [comment]
Good to know!

**Pull request merged**